docs/30DaysOfPWA/advanced-capabilities/04.md

```diff
@@ -4,12 +4,148 @@
 
 ![Placeholder Banner Only. Replace when final assets ready.](_media/day-04.jpg)
 
-### WHAT WE'LL COVER TODAY
-
-| Section | Description |
-| ------- | ----------- |
-| **Definition** | How can I control the way my PWA is displayed? |
-| **Differentiation** | Can I do anything about my PWA window’s titlebar? |
-| **Adoption** | What are some good examples of sites managing how they are displayed? |
-| **Exercise** | Test different display modes. Play with the Window Controls Overlay feature. |
-| **Related** | Look for other OS integrations throughout [Week 2: Advanced Capabilities](../advanced-capabilities). |
+Welcome to day 11 of the [30 Days of PWA](<https://aka.ms/learn-pwa/30Days-blog>) series! Today’s blog post will walk you through the different display modes you can set for your website, and the experimental Window Controls Overlay feature which allows the customization of the title bar of your PWA.
+
+## Display Modes 
+There are four different display modes available to choose from and each provides a different browser UI experience that you’ll want to consider when deciding which display mode is right for your progressive web application. 
+
+You set your preferred display mode via the web app manifest’s display property with one of the following options: `fullscreen`, `standalone`, `minimal-ui`, or `browser`:
+
+```
+{
+
+“display”: “standalone”
+
+}
+```
+
+In the event a browser doesn’t support a display mode, which is possible since they’re not required or support all the display mode options, the display mode will fallback through the following display modes this order: `fullscreen`, `standalone`, `minimal-ui`, and `browser`. 
+
+### Standalone display mode 
+Standalone display mode that makes your PWA look and behave the most like a native application. It opens in a different window from the browser and hides all the browser UI elements like the address bar but keeps standard system UI elements like a back button or a close window button. In this mode, your application can also have its own icon in the application launcher.  If standalone isn’t available, it will fall back to the minimal-ui display mode. Twitter uses `standalone` for their PWA.
```

Comment:
```suggestion
The "standalone" display mode makes your PWA look and behave the most like a native application. It opens in a different window from the browser and hides all the browser UI elements like the address bar. It keeps standard system UI elements like a back button or a close window button. In this mode, your application can also have its own icon in the application launcher.  If standalone isn’t available, it will fall back to the "minimal-ui" display mode. Twitter uses "standalone" for their PWA.
```